src/Dropdown/Dropdown.jsx

```diff
@@ -207,7 +207,8 @@ const Dropdown = ({
           error={error}
           tinted={tinted}
           onClick={toggleDropdown}>
-          {isEmpty ? placeholder : displayValue}
+          {isEmpty && placeholder && <Span>{placeholder}</Span>}
+          {displayValue && <Span>{displayValue}</Span>}

```

Comment:
Please use `TextWrapper` or `SpanWrapper`, just `Span` looks weird to me and it's not something we are using in this codebase.

Reply:
The condition is a little bit trickier than that according to my testing, they had to be in different nodes

I will close this PR for now, I can't reproduce the bug anymore